config/ready_test.go

```diff
@@ -641,26 +641,27 @@ func TestDiffConfigReady(t *testing.T) {
 }
 
 func TestReportConfigReady(t *testing.T) {
-	os.Setenv("GITHUB_EVENT_NAME", "pull_request")
-	os.Setenv("GITHUB_EVENT_PATH", filepath.Join(rootTestdataDir(t), "config", "event_pull_request_opened.json"))
-	os.Setenv("GITHUB_REF", "refs/pull/4/merge")
-	mg := mockedGh(t)
+	t.Setenv("GITHUB_EVENT_NAME", "pull_request")
+	t.Setenv("GITHUB_EVENT_PATH", filepath.Join(rootTestdataDir(t), "config", "event_pull_request_opened.json"))
+	t.Setenv("GITHUB_REF", "refs/pull/4/merge")
+	t.Setenv("GITHUB_TOKEN", "token")
+
 	tests := []struct {
 		c    *Config
 		want string
 	}{
 		{
 			&Config{
 				Repository: "owner/repo",
-				gh:         mg,
+				gh:         mockedGh(t),
```

Comment:
When using the same mockedGh for each test case, the correct response was not returned from the mock server.
Previously, the test passed without problems even when using the common mockedGh because only one test case was making an http request among all the test cases.